packages/std/src/results/attribute.rs

```diff
@@ -9,7 +9,7 @@ pub struct Attribute {
 }
 
 /// Creates a new Attribute.
-pub fn attr<K: ToString, V: ToString>(key: K, value: V) -> Attribute {
+pub fn attr(key: impl ToString, value: impl ToString) -> Attribute {
```

Comment:
Does this change anything or is it just syntactic sugar?

Reply:
Syntactic sugar. I find it a little nicer. I'd be up for updating it everywhere if you're okay with it.

Reply:
In another PR sometime, that is.

Reply:
Alright, thanks for clarification. I have no preference. Feel free to adapt the codebase as you see fit.

Reply:
So it generates the same polymorphic code as the generic version?

I was hoping it would magically do something better under the hood. (Cheap wins to shrink code size are always welcome)

Reply:
AFAIK it does the exact same static dispatch as you'd get with generics, yeah.

Reply:
> Cheap wins to shrink code size are always welcome

This would increase indirection and create runtime burden which is usually not what Rust devs want.